server/services/languagesService.ts

```diff
@@ -52,4 +53,33 @@ export default class LanguagesService {
       user,
     })
   }
+
+  async updateOtherLanguage(
+    clientToken: string,
+    user: PrisonUser,
+    prisonerNumber: string,
+    secondaryLanguageRequest: SecondaryLanguageRequest,
+  ) {
+    const personCommunicationNeedsApiClient = this.personCommunicationNeedsApiClientBuilder(clientToken)
+
+    await personCommunicationNeedsApiClient.updateSecondaryLanguage(prisonerNumber, secondaryLanguageRequest)
+
+    this.metricsService.trackPersonCommunicationNeedsUpdate({
+      fieldsUpdated: ['secondary-language'],
```

Comment:
I think we've got a bit of a mix of kebab case and camel case field names for metrics - would be good to go with one and be consistent